File: Models/Igrac.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    [Table("Igrac")]
    public class Igrac 
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [Range(1000,9999)]
        public int RegBroj { get; set; }

        [Required]
        [RegularExpression("\\w+")]
        [MaxLength(20)]
        public string Ime { get; set; }

        [Required]
        [RegularExpression("\\w+")]
        [MaxLength(20)]
        public string Prezime { get; set; }

        [Required]
        [Range(20,99)]
        public int Starost { get; set; }

        [Required]
        [Range(0,100)]
        public int Ocena { get; set; }

        [Required]
        public int Cena { get; set; }
        
        public Tim Tim { get; set; }

        

    }
}

File: Klijent/Igrac.js
export class Igrac{
    constructor(regbroj, ime, prezime, starost, ocena, cena){
        this.regbroj=regbroj;
        this.ime=ime;
        this.prezime=prezime;
        this.starost=starost;
        this.ocena=ocena;
        this.cena=cena;
        this.container=null;
    }   

    ObrisiIgraca(regbroj) {
        fetch("https://localhost:5001/Igrac/IzbrisatiIgraca/"+regbroj,{
        method:'DELETE'
        })
        .then(s=>{
            if(s.ok)
            {
                alert("Uspesno obrisan igrac "+regbroj);
                location.reload();
            }
            s.text();
        })
    }
    crtajIgraca(host){
        
        
        let red=document.createElement("tr");
        host.appendChild(red);
        this.container=red;

        let el=document.createElement("td");
        el.innerHTML=this.regbroj;
        red.appendChild(el);

        el=document.createElement("td");
        el.innerHTML=this.ime;
        red.appendChild(el);

        el=document.createElement("td");
        el.innerHTML=this.prezime;
        red.appendChild(el);

        el=document.createElement("td");
        el.innerHTML=this.starost;
        red.appendChild(el);

        el=document.createElement("td");
        el.innerHTML=this.ocena;
        red.appendChild(el);

        el=document.createElement("td");
        el.innerHTML=this.cena;
        red.appendChild(el);

        el=document.createElement("button");
        el.innerHTML="Obrisi igraca";
        el.className="btnBrisanje";
        el.name=this.regbroj;
        red.appendChild(el);
        el.onclick=(ev)=>this.ObrisiIgraca(el.name);


    }

} 
    
